utility/fciconv.h

```diff
@@ -60,20 +60,15 @@
     encoding.
 */
 
-#define FC_DEFAULT_DATA_ENCODING "UTF-8"
```

Comment:
Why get rid of this? I think we should use global variables like this instead of hard coding in many places. If something is set in one place, that's one thing. But this is set in more than one spot.

Reply:
I elected to remove duplication instead and merged all `bind_textdomain_codeset` calls into a single location. The use of UTF-8 is baked pretty much all over the place (`QString::fromUtf8`, `qUtf8Printable`) so I don't think one more is a problem.